math_hurdler.py

```diff
@@ -115,6 +117,7 @@ def run(self):
         self.last_answer_index = -1
         question_dirty = True
 
+        pygame.display.init()
```

Comment:
display.init is not needed when PygameCanvas is called with modules argument, see sugargame/test activity.